File: PTH/pth.py
#This program gives TLE for large input outputs, of order 10^17-18. Hence a corresponding C program was written to achieve the same in half the time required by this program.

from math import *
t=int(input())
for i in range(t):
	n=int(input())
	x=sqrt(1+n)-1
	print('Case ',i+1,': ',int(x),sep='');

File: PTH/PTH.c
#include<stdio.h>
#include<math.h>
long long int n;
int main()
{
    int t,i;
    scanf("%d",&t);
    for(i=1;i<=t;i++)
    {
                     scanf("%lld",&n);
                     long long int x=(long long int)sqrt(1+n);
                     printf("Case %d: %lld\n",i,x-1);
    }
}
